osd_cluster_create/byo-vpc.adoc

```diff
@@ -0,0 +1,35 @@
+[id="byo-vpc"]
+= Managing a Bring-Your-Own Virtual Private Cloud (VPC)
+:context: byo-vpc
+include::modules/attributes-openshift-dedicated.adoc[]
+
+toc::[]
+
+Configuring a cluster to include your own pre-configured Virtual Private Cloud (VPC) allows for the use of any existing network that relies on an existing VPC.
+
+.Prerequisites
+
+* You have a configured VPC. Use the following documentation for guidance on how to configure the VPC's components, such as:
+** a private subnet
+** a public subnet
+** a NAT gateway
+** an Internet gateway
+* You have a configured AWS account in your AWS organization.
+
+.Procedure
+
+. Create an IAM user with the name `osdCcsAdmin`.
+.. On this role, attach the `AdministratorAccess` policy.
+.. Generate an access key pair.
+.. Store this generated key pair for use later in this process.
+. Once you have the AWS account ID, you are ready to add a pre-configured VPC.
+
+include::modules/byo-vpc-prep.adoc[leveloffset=+1]
+
+include::modules/byo-vpc-single-az.adoc[leveloffset=+1]
+
+include::modules/byo-vpc-multiple-az.adoc[leveloffset=+1]
+
+.Additional Resources
+
+* See xref:../osd_quickstart/osd-quickstart.adoc[OSD Quick Start] for general cluster creation information.
```

Comment:
* xrefs should always contain an anchor (#<anchorname>) after the `.adoc`
* Also, the title isn't actually "OSD Quick Start", it's "Quick start", so I'd use that.